src/types/Account/_sortPlugins.js

```diff
@@ -0,0 +1,140 @@
+const { each, findIndex } = require('lodash');
+const TransactionSyncStreamWorker = require('../../plugins/Workers/TransactionSyncStreamWorker/TransactionSyncStreamWorker');
+const ChainPlugin = require('../../plugins/Plugins/ChainPlugin');
+const IdentitySyncWorker = require('../../plugins/Workers/IdentitySyncWorker');
+const { WALLET_TYPES } = require('../../CONSTANTS');
+
+const initPlugin = (UnsafePlugin) => {
+  const isInit = !(typeof UnsafePlugin === 'function');
+  return (isInit) ? UnsafePlugin : new UnsafePlugin();
+};
+
+/**
+ * Sort user defined plugins using the injectionOrder properties before or after when specified.
+ *
+ * Except if specified using before property, all system plugins (TxSyncStream, IdentitySync...)
+ * will be sorted on top.
+ *
+ * @param defaultSortedPlugins
+ * @param userUnsafePlugins
+ * @returns {*[]}
+ */
+const sortUserPlugins = (defaultSortedPlugins, userUnsafePlugins, allowSensitiveOperations) => {
+  const sortedPlugins = [];
+  const initializedSortedPlugins = [];
+
+  // We start by ensuring all default plugins get loaded and initialized on top
+  defaultSortedPlugins.forEach((defaultPluginParams) => {
+    sortedPlugins.push(defaultPluginParams);
+
+    // We also need to initialize them so we actually as we gonna need to read some properties.
+    const plugin = initPlugin(defaultPluginParams[0]);
+    initializedSortedPlugins.push(plugin);
+  });
+
+  // Iterate accross all user defined plugins
+  each(userUnsafePlugins, (UnsafePlugin) => {
+    const plugin = initPlugin(UnsafePlugin);
+
+    const {
+      awaitOnInjection,
```

Comment:
I still don't get where is this option is using actually ;)

Reply:
ChainPlugin, IdentitySyncWorker, TransactionSyncStreamWorker use them. 
This is required so that DashPayWorker only gets injected after the injection of IdentitySyncWorker